DES: Added DESController.java

PuravUdayDesai opened the pull request:
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue Number: #1553

PuravUdayDesai committed:
Merge pull request #1 from TesseractCoding/master

Sync Master Repository to Forked Repository

PuravUdayDesai committed:
Added DESController.java

PuravUdayDesai committed:
Moved DESController.java under des/

PuravUdayDesai commented:
@HarshCasper please review the pull request once, if any changes/improvements/modifications needed please let me know about the respective.

Sequence Number: 01

Issue Number: #1553 
Pull Request Number:  #2078

HarshCasper commented on Java/cryptography/des/DESController.java:
```diff
@@ -0,0 +1,153 @@
+	 * </ul>
+	 * 
+	 * @param args not used
+	 */
```
What is the purpose of using these tags here?

PuravUdayDesai replied:
It is used maintain a javadoc, it just tells which all methods and objects are used in main method, if anything incorrect, please do let me now.

HarshCasper replied:
Please remove them up. We are not maintaining a Javadoc here

PuravUdayDesai replied:
Okay @HarshCasper I'll do that

HarshCasper commented on Java/cryptography/des/DESController.java:
```diff
@@ -0,0 +1,153 @@
+		}
+		
+	}// End of main(String[]) method
+}// End of class DESController
```
Redundant Comments

PuravUdayDesai replied:
What shall I do for it please guide

HarshCasper replied:
Remove them up

PuravUdayDesai replied:
Ok @HarshCasper I'll do that

HarshCasper commented on Java/cryptography/des/DESController.java:
```diff
@@ -0,0 +1,153 @@
+		{
+			//used for styling the output
+			if (i % 8 == 0 && i != 0)
+			{
```
Can the Core Logic of the Code shift into some other function? We should ideally use the main() function only for the Driver Code

PuravUdayDesai replied:
Okay I understand your concern I'll do that for sure, really thank you for guidance.

HarshCasper reviewed (changes requested):
Please make apt README changes as well

PuravUdayDesai commented:
> Please make apt README changes as well

@HarshCasper Please elaborate the incorrect things you observed in the pull request and the changes you're looking forward.

HarshCasper commented:
> > Please make apt README changes as well
> 
> @HarshCasper Please elaborate the incorrect things you observed in the pull request and the changes you're looking forward.

I have already added my comments. Please resolve them to continue forward. Also take a look at the merged PRs and have an insight into the entire workflow!

PuravUdayDesai commented:
> > > Please make apt README changes as well
> > 
> > 
> > @HarshCasper Please elaborate the incorrect things you observed in the pull request and the changes you're looking forward.
> 
> I have already added my comments. Please resolve them to continue forward. Also take a look at the merged PRs and have an insight into the entire workflow!

Ok @HarshCasper Thank you for your comments, I'll update them accordingly and revert

PuravUdayDesai committed:
Updated DESController.java

Reduced redundant code for printing the plain text, cipher text and decrypted text.

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai commented:
@HarshCasper I have made the told changes in each and every file in which I observed the given issues.

PuravUdayDesai commented:
@atarax665 please reopen the closed issue, I have completed the given changes, sorry for any delay.

HarshCasper commented:
Closing this Issue because of no updates.

HarshCasper commented:
Closing this PR because of no updates.

PuravUdayDesai commented:
@HarshCasper I had submitted the updates which you requested 12 days ago, for which other updates you are looking for?
